script/testing/junit/run_junit.py

```diff
@@ -79,6 +79,8 @@ def _wait_for_db_server(self):
             except:
                 time.sleep(0.1)
                 continue
+        self._stop_db_server()
```

Comment:
Do you need to do this if it failed to start? Probably doesn't matter.

Reply:
We need to terminate the child process for DB server in case of timeout (not failure though), otherwise the next time we will get a complaint of address already in use